source/MaterialXGenShader/OCIOColorManagementSystem.cpp

```diff
@@ -403,6 +632,13 @@ ImplementationPtr OCIOColorManagementSystem::getImplementation(const ColorSpaceT
         // Retrieve information
         gpu->extractGpuShaderInfo(shaderDesc);
 
+#if _UNDEFINED_SYMBOLS_IF_INCLUDE_
```

Comment:
I cannot add comments on unchanged lines, so here are my comments:
- when there is a problem OpenColorIO throws, so `if(!processor)`, `if(!gpu)` and `if(!shaderDesc)` are useless.
- line 636 to line 640 are replacing line 616 to line 633